dagmc_stats/DagmcQuery.py

```diff
@@ -7,61 +7,88 @@
 
 class DagmcQuery:
     def __init__(self, dagmc_file, meshset=None):
-        """Constructor
+        """This class provides the functionality for making queries about
+        various metrics for the meshset(s) of interest.
 
         inputs
         ------
-        dagmc_file : DagmcFile instance
-        meshset: the meshset on which query will be performed. The rootset will be used by default.
+            dagmc_file : DagmcFile instance
+            meshset: the meshset on which query will be performed.
+                The rootset will be used by default.
 
         outputs
         -------
-        none
+            none
         """
         self.dagmc_file = dagmc_file
         self.meshset_lst = []
         if meshset is None:
-            self.meshset_lst.append(self.dagmc_file.root_set)
+            self.meshset_lst =  self.dagmc_file.entityset_ranges['surfaces']
         else:
             self.__get_entities(meshset)
+        self.tris = self.__get_tris()
+        self.verts = self.__get_verts()
```

Comment:
These three now have a slightly different pattern.  `__get_entities()` just sets the member variable internally and returns nothing.  The other two return data and the member variable is set here.

For maintainability, it probably makes sense for them to be similar.